test/spec/helper.pages.js

```diff
@@ -34,11 +34,16 @@ test( "previousPage should decrement the current page by one", function( t ) {
 } );
 
 test( "pages should be reset if the mutation might change the number of pages", function( t ) {
+  var bindAll = require( "lodash/function/bindAll" );
+  var partial = require( "lodash/function/partial" );
+
   var helper = algoliasearchHelper( "", "", {
     facets : [ "facet1", "f2" ],
     disjunctiveFacets : [ "f1" ]
   } );
 
+  bindAll(helper);
```

Comment:
That's a bit invasive. Aren't there cases, where this could make something go wrong?